test/components/ChartForm_test.js

```diff
@@ -431,12 +397,7 @@ describe('ChartForm.vue', () => {
     it('creates the expected date field options for the selected event', () => {
       const dateFieldOptions = wrapper.findAll(`${selector.dateFieldSelect} > option`);
       expect(dateFieldOptions.length).toEqual(2); // 1 options plus prompt
-      expect(
-        dateFieldOptions
-          .at(1)
-          .text()
-          .trim()
-      ).toEqual('survey_date (Survey date)'); // sorted by label
+      expect(dateFieldOptions.at(1).text().trim()).toEqual('survey_date (Survey date)'); // sorted by label
```

Comment:
Given that this is all on a single line now it might read better to put the comment above the code.

Reply:
Good point. I just glanced over the code after running the formatter, so I'll adjust this and look for anything else that could be improved.